package/opt/hassbian/suites/hassbian-script.sh

```diff
@@ -12,68 +12,38 @@ function hassbian-script-show-copyright-info {
 }
 
 function hassbian-script-upgrade-package {
-
 if [ "$DEV" == "true"  ]; then
   echo "This script downloads new scripts directly from the dev branch on Github."
   echo "you can use this to be on the 'bleeding edge of the development of Hassbian.'"
   echo "This is not recommended for daily use."
   echo -n "Are you really sure you want to continue? [N/y] : "
   read -r RESPONSE
   if [ "$RESPONSE" == "y" ] || [ "$RESPONSE" == "Y" ]; then
-    RESPONSE="Y"
+    devbranch="-dev"
   else
     echo "Exiting..."
     return 0
   fi
-  echo "Creating and changing in to a temporary folder."
-  cd || exit
-  sudo mkdir /tmp/hassbian_config_update
-  cd /tmp/hassbian_config_update || exit
-
-  echo "Downloading new scripts from github."
-  curl -L https://api.github.com/repos/home-assistant/hassbian-scripts/tarball| sudo tar xz --strip=1
-
-  echo "Moving scripts to the install folder."
-  yes | sudo cp -rf /tmp/hassbian_config_update/package/usr/local/bin/hassbian-config /usr/local/bin/hassbian-config
-  yes | sudo cp -rf /tmp/hassbian_config_update/package/opt/hassbian/suites/* /opt/hassbian/suites/
-
-  echo "Removing the temporary folder."
-  cd || exit
-  sudo rm -r /tmp/hassbian_config_update
+fi
+echo "Updating apt information..."
+echo "deb [trusted=yes] https://gitlab.com/hassbian/repository$devbranch/raw/master stretch main" | sudo tee /etc/apt/sources.list.d/hassbian.list
+apt update
+
+echo "Checking installed version..."
+current_version=$(apt list hassbian-scripts | tail -1 | awk -F'[' '{print $NF}' |  awk -F']' '{print $1}')
+if [ "$current_version" != "installed" ]; then
+  echo "Removing old version of hassbian-scripts..."
+  apt purge -y hassbian-scripts
+  apt clean
+
+  echo "Installing newest version of hassbian-scripts..."
+  echo "deb [trusted=yes] https://gitlab.com/hassbian/repository$devbranch/raw/master stretch main" | sudo tee /etc/apt/sources.list.d/hassbian.list
```

Comment:
Shouldn't a check be run before to check if this file exists?

Reply:
No need :)
If it do not exist it will be created :)